json-decode.go

```diff
@@ -362,7 +362,7 @@ func (cdc *Codec) decodeReflectJSONSlice(bz []byte, info *TypeInfo, rv reflect.V
 }
 
 // CONTRACT: rv.CanAddr() is true.
-func (cdc *Codec) decodeReflectJSONStruct(bz []byte, info *TypeInfo, rv reflect.Value, fopts FieldOptions) (err error) {
+func (cdc *Codec) decodeReflectJSONStruct(bz []byte, info *TypeInfo, rv reflect.Value, _ FieldOptions) (err error) {
```

Comment:
is it possible to remove the last parameter? don't see a reason to keep it

Reply:
yes it's possible. i didn't remove it because the binary version has it:
https://github.com/tendermint/go-amino/blob/59d50ef176f686dc2c0c150e85a9381a78b9978a/binary-decode.go#L763-L764

Reply:
updated 👍